__tests__/app.test.js

```diff
@@ -181,3 +189,29 @@ describe('GET /api/articles/:article_id/comments', () => {
         })
     })
 })
+
+describe('GET /api/users', () => {
+    test('status:200, should return an array of user objects, with username, name and avatar_url properties', () => {
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({body}) => {
+            const { users } = body
+                expect(users).toHaveLength(4)
+            users.forEach((user) => {
+                expect(user).toHaveProperty('username', expect.any(String))
+                expect(user).toHaveProperty('name', expect.any(String))
+                expect(user).toHaveProperty('avatar_url', expect.any(String))
+            })
+        })
+    })
+    test('status:404, should return an error msg Not found when passed a route that does not exist', () => {
```

Comment:
You've already done this way up around line 30, so I wouldn't worry about checking it again :)